src/jit/compiler.cpp

```diff
@@ -2011,16 +1992,14 @@ void                Compiler::compInitOptions(CORJIT_FLAGS* jitFlags)
 
 #else // !DEBUG
 
-    LPWSTR altJitVal;
+    const char* altJitVal;
     if (opts.eeFlags & CORJIT_FLG_PREJIT)
     {
-        static ConfigString fAltJitNgen;
-        altJitVal = fAltJitNgen.val(CLRConfig::INTERNAL_AltJitNgen);
+        altJitVal = JitConfig.AltJitNgen().list();
```

Comment:
Maybe I'll figure it out later, but why does this one use .list() when other ConfigString replacements I've seen don't?